config/v1/zz_generated.crd-manifests/0000_10_config-operator_01_schedulers-Default.crd.yaml

```diff
@@ -1,123 +0,0 @@
-                  impacted even if this field is set. This adds an annotation section to
-                  the namespace.
-                  For example, if a new namespace is created with
-                  node-selector='type=user-node,region=east',
-                  the annotation openshift.io/node-selector: type=user-node,region=east
-                  gets added to the project. When the openshift.io/node-selector annotation
-                  is set on the project the value is used in preference to the value we are setting
-                  for defaultNodeSelector field.
-                  For instance,
-                  openshift.io/node-selector: "type=user-node,region=west" means
-                  that the default of "type=user-node,region=east" set in defaultNodeSelector
-                  would not be applied.
-                type: string
-              mastersSchedulable:
-                description: |-
-                  mastersSchedulable allows masters nodes to be schedulable. When this flag is
-                  turned on, all the master nodes in the cluster will be made schedulable,
-                  so that workload pods can run on them. The default value for this field is false,
-                  meaning none of the master nodes are schedulable.
-                  Important Note: Once the workload pods start running on the master nodes,
-                  extreme care must be taken to ensure that cluster-critical control plane components
-                  are not impacted.
-                  Please turn on this field after doing due diligence.
-                type: boolean
-              policy:
-                description: |-
-                  DEPRECATED: the scheduler Policy API has been deprecated and will be removed in a future release.
-                  policy is a reference to a ConfigMap containing scheduler policy which has
-                  user specified predicates and priorities. If this ConfigMap is not available
-                  scheduler will default to use DefaultAlgorithmProvider.
-                  The namespace for this configmap is openshift-config.
-                properties:
-                  name:
-                    description: name is the metadata.name of the referenced config
-                      map
-                    type: string
-                required:
-                - name
-                type: object
-              profile:
-                description: |-
-                  profile sets which scheduling profile should be set in order to configure scheduling
-                  decisions for new pods.
-
-                  Valid values are "LowNodeUtilization", "HighNodeUtilization", "NoScoring"
-                  Defaults to "LowNodeUtilization"
-                enum:
-                - ""
-                - LowNodeUtilization
-                - HighNodeUtilization
-                - NoScoring
```

Comment:
they don't seem to be related to DynamicResourceAllocation, is it safe for these to be deleted? does it have any impact on the upgrade? cc @ingvagabund